core/node/api_server/src/web3/namespaces/en.rs

```diff
@@ -90,17 +90,18 @@ impl EnNamespace {
         let mut storage = self.state.acquire_connection().await?;
         let genesis_batch = storage
             .blocks_dal()
-            .get_storage_l1_batch(L1BatchNumber(0))
+            .get_l1_batch_metadata(L1BatchNumber(0))
```

Comment:
but why are you changing it? it's more db requests

Reply:
- I'm changing it because storage models leak through the abstraction layer – DAL – intended to prevent such leaking.
- Having storage models be public obscures their flaws such as unused fields. Note that I've removed 2 unused and one almost unused fields from `StorageL1Batch` and one more unused field from `StorageL1BatchHeader`.
- It could be a single DB request if the new `get_optional_l1_batch_metadata()` method is used. Does the genesis batch have all metadata, or can it be incomplete?
- As the comment at the start of the method states, "[i]f this method will cause some load, we can cache everything in memory", so it's not like performance is a big concern here anyway.